src/commands/UnbanBanCommand.ts

```diff
@@ -51,8 +51,13 @@ export async function parseArguments(
     let list: PolicyList | null = null;
     let force = false;
     while (argumentIndex < 7 && argumentIndex < parts.length) {
-        const arg = parts[argumentIndex++];
+        let arg = parts[argumentIndex++];
         if (!arg) break;
+        if (arg.startsWith("https://matrix.to/#/")) {
+            // it's a matrix.to link, parse it
+            const components = arg.split("https://matrix.to/#/");
+            arg = components[1];
+        }
```

Comment:
`matrix-bot-sdk` has utilities for processing permalinks, namely [`Permalinks.parseUrl`](https://github.com/element-hq/matrix-bot-sdk/blob/1f8c29bafca57dcf71cd6900891cb5227db0cc7c/src/helpers/Permalinks.ts#L79) which should handle all the weird edge cases too. 

We'd need to try/catch the parsing, and probably assume that any error means it's not a matrix.to URI